README.md

```diff
@@ -29,14 +29,7 @@ Because of a deep desire to develop [client diversity](https://clientdiversity.o
 ### Install dependencies
 
 ```
-sudo apt update && sudo apt install -y golang-go curl git default-jre make gcc python3-distutils-extra python3-setuptools
-```
-```
-curl -fsSL https://deb.nodesource.com/setup_20.x | sudo -E bash -
-sudo apt install -y nodejs
-```
-```
-sudo npm install --global yarn
+sudo apt update && sudo apt install -y golang-go curl git default-jre
```

Comment:
I honestly don't even know why any of these besides `curl` would be needed but removed the heavy libraries which I assume were just needed to make Lodestar install work